packages/phoenix-evals/src/phoenix/evals/preview/evaluators.py

```diff
@@ -613,3 +609,143 @@ def create_classifier(
         required_fields=required_fields,
         direction=direction,
     )
+
+
+def evaluate_dataframe(
+    dataframe: pd.DataFrame,
+    evaluators: List[Evaluator],
+    input_mappings: Optional[Dict[str, Mapping[str, str]]] = None,
```

Comment:
I feel like we might want to define mappings on the evaluators themselves? This way we can generically help users map evaluator inputs to the template variables in other methods too.

I do wonder if we want to allow users to pass in a mapping of *lambdas* here that can transform the dataframe before being passed into each evaluator:

as pseudocode:

```
for e in evaluators:
    transformed_df = transforms[e.name](dataframe.copy())
    # evaluate each row of transformed_df
```